uiowa/uiowa
# Add field_person_department to genetics.grad.uiowa.edu site split

Resolves #4021 
Resolves #4037 





# How to test



`blt ds --site=genetics.grad.uiowa.edu`
Create or edit a person
Add one or more departments
Check that you can add one or more advisors
Visit https://gradgenetics.local.drupal.uiowa.edu/people/students and verify that education and hometown columns are showing
Visit https://gradgenetics.local.drupal.uiowa.edu/people/students/former and verify that started column is showing (there is no data yet)

`blt ds --site=immuno.grad.uiowa.edu`
Create or edit a person
Check that you can add more than one department.
Check that you can add more than one advisor
